clients/web/src/utilities/EventStream.js

```diff
@@ -64,6 +73,7 @@ prototype.open = function () {
                 stream.trigger('g:error', e);
                 return;
             }
+            window.localStorage.setItem('sseTimestamp', obj._girderTime);
```

Comment:
There are pros and cons to both.  Unless it is something we want transmitted on most requests or needs some sort of security, then localStorage would be preferred, I think.  Cookies get transmitted on every request.  localStorage is supported by all modern browsers (and if we keep track of the value in a local variable as well, then when it isn't, such as in private mode, we just fail to remember state when we reload the whole page).